mvcc/kvstore_test.go

```diff
@@ -513,6 +513,52 @@ func TestRestoreContinueUnfinishedCompaction(t *testing.T) {
 	t.Errorf("key for rev %+v still exists, want deleted", bytesToRev(revbytes))
 }
 
+func TestRestoreContinueUnfinishedCompaction2(t *testing.T) {
```

Comment:
can this test pass without the fix? 

we need to rename this test to something more meaningful instead of just adding `2`.

Reply:
this test cannot be passed, it triggers the deadlock in `Restore` function